blocks/cards/cards.css

```
@@ -429,7 +429,8 @@
     }
   
     main .cards-container .default-content-wrapper h3 {
-      font-size: 28px;
+      font-size: 24px;
+      margin-top:35px;
```

use consistent style
```suggestion
      margin-top: 35px;
```